install_on_linux.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+COMPOSE_SWITCH_VERSION="1.0.0"
+
+# FIXME Use final URLs
+COMPOSE_SWITCH_URL="https://github.com/docker/compose-switch/releases/download/$COMPOSE_SWITCH_VERSION/docker-compose"
+
+DOCKER_COMPOSE_V1_PATH_ORIG="$(which docker-compose)"
+DOCKER_COMPOSE_V1_PATH_NEW="$DOCKER_COMPOSE_V1_PATH_ORIG-v1"
+
+if [ ! -f "${DOCKER_COMPOSE_V1_PATH_ORIG}" ]; then
+  echo "'docker-compose' version 1 could not be found in PATH. Aborting..."
```

Comment:
Everywhere else we use ` v1 ` , what about
```suggestion
  echo "'docker-compose' v1 could not be found in PATH. Aborting..."
```